src/app/playlists/page.tsx

```
@@ -59,12 +96,21 @@ export default function PlaylistsPage() {
+            <h1 className="text-3xl font-bold tracking-tight">ライブラリ</h1>
+            <p className="text-zinc-400 text-sm mt-1">
+              保存した曲やスキップした曲
+            </p>
+          </div>
+          <button
+            onClick={() => setIsModalOpen(true)}
+            className="p-2 bg-white text-black rounded-full hover:bg-zinc-200 transition-colors"
+            aria-label="Create Playlist"
+          >
+            <Plus className="h-6 w-6" />
+          </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)